app/elements/countdown-timer/countdown-timer.html

```diff
@@ -143,9 +143,47 @@
         attached: function() {
           this.countdownController = new IOWA.CountdownTimer.Core(new Date(this.date), Polymer.dom(this).node);
           this.countdownController.showCurrentTime = this.showCurrentTime;
-          this.countdownController.setUp(_skipIntro);
+          this.countdownController.setUp(this.showCurrentTime || _skipIntro);
           this.countdownController.attachEvents();
 
+          if (this.showCurrentTime) {
+            this.logoController = new IOWA.CountdownTimer.Core(new Date(this.date), Polymer.dom(this).node);
+            this.logoController.canvasElement.style.position = 'absolute';
+            this.logoController.canvasElement.style.top = '0px';
+            this.logoController.canvasElement.style.left = '0px';
+
+            TweenMax.set(this.countdownController.canvasElement, { autoAlpha: 0 });
+
+            var node = Polymer.dom(this).node;
+
+            node.addEventListener('mouseover', function() {
+              this.countdownController.play();
+              TweenMax.to(this.countdownController.canvasElement, 0.3, { autoAlpha: 1 });
+              TweenMax.to(this.logoController.canvasElement, 0.3, {
+                autoAlpha: 0,
+                onComplete: function() {
+                  this.logoController.pause();
+                },
+                onCompleteScope: this
+              });
+            });
+
+            node.addEventListener('mouseout', function() {
```

Comment:
Actually, it might be easier to use `this.listen`/`this.unlisten` or `listeners`: https://www.polymer-project.org/1.0/docs/devguide/events#imperative-listeners